frontend/src/views/ProfileView.vue

```diff
@@ -49,7 +49,7 @@ const editProfile = async () => {
     <div class="text-4xl font-light">{{ user.data.name }}</div>
     <div class="flex items-center gap-4">
       <div class="rounded-full border border-gray-500 px-2 text-xs uppercase text-gray-500">
-        {{ keyByValue(subscriberLevels, user.data.level) }}
+        {{ keyByValue(SubscriberLevels, user.data.level, true) }}
```

Comment:
Although this is not part of the current PR, I noticed:
`keyByValue` is using an optional boolean parameter. It's hard to understand what this is doing when reading the implementation. 

This can be fixed with an object parameter with the options you want to pass. In this case having a third parameter as `{ isEnum: boolean }` would be easier to read.

Then you can have:
`keyByValue(SubscriberLevels, user.data.level, { isEnum: true })`

Reply:
Thank you for this suggestion. For your proposed solution I would need to change the function signature though, otherwise TS will complain about an argument of type '{ isEnum: boolean; }' not being assignable to parameter of type 'boolean'. I'll take a note to type this kind of function with an own signature and improve that after the heavy TS work is done. For now, your IDE will reveal the boolean argument name when typing or hovering that function 😇